code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -891,7 +677,7 @@
 	mag_type = MAGAZINE
 
 /obj/item/ammo_casing/stg
-	desc = "A 7.92×33mm Kurz casing."
+	desc = "A 7.92 33mm Kurz casing."
 	icon_state = "rifle-casing"
 	caliber = "kurz"
 	projectile_type = /obj/item/projectile/bullet/rifle/a762
```

Comment:
This looks incorrect. Nothing to do with this PR but this is probably something that should be fixed later.